kinbot/mess.py

```diff
@@ -1,17 +1,27 @@
-from __future__ import division
+from __future__ import print_function
+import logging
 import os
 import subprocess
 import time
 import pkg_resources
-import logging
+
 from kinbot import constants
 from kinbot import frequencies
-from kinbot import cheminfo
+from kinbot.uncertaintyAnalysis import UQ
+
 
 class MESS:
     """
-    Class that reads and write all MESS files and runs MESS
+    Class that read and writes MESS files for UQ analysis
+    Code is the same as mess.py exceppt that it loops to create 'n' files
```

Comment:
What do you mean that it's the same as mess.py? This is mess.py.

Reply:
I forgot to rewrite this. Originally, when I first started UQ, I had a separate UQ_mess.py file. But I merged that with the mess.py file to avoid being redundant.